server/service/osquery_utils/queries.go

```diff
@@ -353,6 +353,11 @@ var extraDetailQueries = map[string]DetailQuery{
 		// always present.
 	},
 	OrbitInfoQueryName: OrbitInfoDetailQuery,
+	"kubequery_info": {
+		Query:      `SELECT * from kubernetes_info`,
+		IngestFunc: ingestKubequeryInfo,
+		Discovery:  discoveryTable("kubernetes_info"),
+	},
```

Comment:
Given that this new ingestion updates `hosts` aka "not directly" (it doesn't update secondary tables like all the other `extraDetailQueries`), it should be defined in `hostDetailQueries` instead.

We should eventually enforce this properly. Currently it's just documented...
```go
// hostDetailQueries defines the detail queries that should be run on the host, as
// well as how the results of those queries should be ingested into the
// fleet.Host data model (via IngestFunc).
//
// This map should not be modified at runtime.
var hostDetailQueries = map[string]DetailQuery{

[...]

// extraDetailQueries defines extra detail queries that should be run on the host, as
// well as how the results of those queries should be ingested into the hosts related tables
// (via DirectIngestFunc).
//
// This map should not be modified at runtime.
var extraDetailQueries = map[string]DetailQuery{
```